### PR TITLE
Convert to pure plugin2

### PR DESCRIPTION
**Pull request by SysPete**
I don't know how soon we want to start moving all plugins over to plugin2 but I thought I may as well make a start.

### CODE REVIEW

File: lib/Dancer2/Plugin/Ajax.pm [hunk 1]
```diff
@@ -3,49 +3,59 @@ package Dancer2::Plugin::Ajax;
...
 
-    my $content_type = plugin_setting->{content_type} || 'text/xml';
-
     my $ajax_route = sub {
 
         # # must be an XMLHttpRequest
-        if ( not $dsl->app->request->is_ajax ) {
-            $dsl->pass and return 0;
+        if ( not $plugin->app->request->is_ajax ) {
+            $plugin->app->pass and return 0;
```
Comment (SysPete) [inline lib/Dancer2/Plugin/Ajax.pm]: I guess the `return 0` here is superfluous.

**Comment by SysPete** [comment]
@bigpresh @cromedome @xsawyerx any chance of a release?

**Comment by cromedome** [comment]
Let me check to see if I have the correct rights on PAUSE for this. I'll be happy to release if so.

**Comment by cromedome** [comment]
:+1: btw

**Comment by racke** [comment]
Yeah let's go ahead :+1:  :jack_o_lantern:

**Comment by cromedome** [comment]
Merged and released! On its way to CPAN.

**Pull request merged**